coalib/output/ConsoleInteraction.py

```diff
@@ -309,6 +309,8 @@ def print_results_formatted(log_printer,
                       'column:{column}:end_line:{end_line}:end_column:'
                       '{end_column}:severity:{severity}:severity_str:'
                       '{severity_str}:message:{message}')
+    file_dict = args[0]
```

Comment:
https://github.com/coala/coala/pull/3386#discussion_r94294547 you gotta add this to the params, this isn't well readable.